src/main.zig

```diff
@@ -643,6 +642,10 @@ pub fn main() void { // MARK: main()
 		lastBeginRendering = begin;
 
 		Window.handleEvents(deltaTime);
+		if (Window.Gamepad.areControllerMappingsDownloading() and !gui.isWindowOpen("download_controller_mappings")) {
```

Comment:
Why not just open the window when you start downloading?

Reply:
I didn't want to add a dependency on the GUI to the Window.Gamepad

Reply:
I didn't realize it already had access to the GUI, through an existing import